spec/integ/crypto/crypto.spec.ts

```diff
@@ -2202,9 +2202,11 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
                     "express:/_matrix/client/v3/user/:userId/account_data/:type(m.secret_storage.*)",
                     (url: string, options: RequestInit) => {
                         const content = JSON.parse(options.body as string);
+
                         if (content.key) {
                             resolve(content.key);
                         }
+
```

Comment:
not convinced this is an improvement, but ymmv

Reply:
not sure I touched that, I am a bit confused